frameworklauncher/conf/frameworklauncher.yml

```diff
@@ -3,6 +3,9 @@
 zkConnectString: 127.0.0.1:2181
 zkRootDir: /Launcher
 hdfsRootDir: /Launcher
+rootAdminUsers: !!set
+  ? {name: ra1}
+  ? {name: ra2}
```

Comment:
Better change it to a real example like:
"rootAdminUsers" : [ {
      "name" : "admin1"
    }, {
      "name" : "admin2"
    } ],

Reply:
OK

---
In reply to: [170492468](https://github.com/Microsoft/pai/pull/320#discussion_r170492468) [](ancestors = 170492468)